docs/osf-trivia-list/dasharo.md

```diff
@@ -0,0 +1,220 @@
+FSP, ME, GbE etc.).
+
+In coreboot community there was some controversy about calling Dasharo
+open-source firmware distribution (for details please check [gerrit review][gerrit-review].
+We respect coreboot community opinion, so we agreed that in case of coreboot
+documentation it would be better to use open-source based firmware
+distribution. It doesn't mean we agree with that decision:
+* Definitions and rules used in coreboot documentation review
+* Rules seem to no be applied equally to all contributors of [coreboot
+distribution][cb-distro]
+
+## Why Dasharo is not called coreboot firmware distribution?
+
+Dasharo is not coreboot firmware distribution because we also may imagine
+Dasharo pure EDKII, oreboot or based on any other open-source firmware
+framework. Dasharo typically is based on coreboot and EDKII, plain coreboot is
+not enough in most cases for booting modern computer.
+
+## What value Dasharo provide in comparison to coreboot?
+
+* Dasharo is open-source firmware distribution based on coreboot and other
+open-source firmware frameworks, you can think about Dasharo and coreboot
+relation in the same way as you think about Debian/Ubuntu/RedHat and Linux.
+* We provide long term maintenance - coreboot community for various reasons, do
+not merge some patches, because understaffing, lack of reviewer some changes
+have long way to upstream, we maintain those patches and make them work
+before those will go upstream, if ever, we are committed to maintain platforms
+which are moved to branch in coreboot.
+* We provide transparent validation results - coreboot in itself provide no
+guarantees around release quality and do not provide binary distribution (for
+reference please check [coreboot project scope][coreboot-scope], we provide
+those in scope of validation we perform.
+* We provide seamless deployment - our documentation provide information about
+initial deployment, updates and recovery procedures. We working on commercial
+product called [Dasharo Tools Suite OS][dtsos], which will largely improve firmware
+deployment, hardware compatibility reporting and  binary blobs transmission
+wherever it is necessary, as well as recovery.
+* Firmware update - we are registered [consultants for fwupd/LVFS][lvfs] and enable
+customers and community platforms, so they can get seamless firmware update in
+Linux.
+* Support  - Dasharo team is paid to provide support in community through
+[Dasharo Matrix Space][dasharo-matrix], of course we want to introduce some
+commercial products related to support to make business feasible, but some
+level of support always would be available to community.
+* We provide ready to use binaries with GPG based signing scheme that improve
+verification where firmware coming from.
+
+## What are the differences between the official coreboot repository and the Dasharo repository ?
+
+The difference is support for [compatible
+hardware](https://docs.dasharo.com/#compatible-hardware). Dasharo repository
```

Comment:
```suggestion
The difference is support for [compatible hardware](https://docs.dasharo.com/#compatible-hardware). 
Dasharo repository...
```

The links should not be broken

Reply:
pre-commit accept that and mkdocs render that correctly, so I guess this rule may be no longer valid